js/infobar.js

```diff
@@ -253,6 +270,18 @@
     InfoBar.prototype.oncancel = {};
 
     /**
+     * Attempts to store the disabled status of the specified InfoBar in sessionStorage
+     * @param {string} prefName - The stored preference name for this InfoBar
+     */
+    InfoBar.prototype.setDisabledStatus = function(prefName) {
+        try {
+            sessionStorage.setItem(prefName, 'true');
+        } catch (ex) {
+            console.log('oncancel: Error while setting didabled status of InfoBar in sessionStorage');
```

Comment:
Sigh, yeah, client-side error handling isn't exactly robust. For serious errors, could do an AJAX POST to a back-end service that alerts devs, but that seems overkill here. At least in this case, it's an error due to lack of browser support and not an error in the code.